int16[4] amk_actual_velocity
int16[4] amk_torque_current
int16[4] amk_magnetizing_current
int16[4] amk_temp_motor
int16[4] amk_temp_inverter
uint16[4] amk_error_info
int16[4] amk_temp_igbt
